raster/src/main/scala/geotrellis/raster/Tile.scala

```diff
@@ -292,4 +292,10 @@ trait Tile extends CellGrid with IterableTile with MappableTile[Tile] {
     }
     s
   }
+
+  def isNoDataTile = {
+    var empty = true
+    foreach { z => if (isData(z)) empty = false }
```

Comment:
The much faster way to do this is to have a return inside of a while loop, or some other way to break immediately upon finding a non-nodata value